data/absolutely-positioned-inline-element-shifted.yml

```diff
@@ -0,0 +1,31 @@
+title: "Absolutely positioned inline-block (or inline) element with floating nextSibling
+  and auto 'left' is positioned to the right of the float"
+
+severity: normal
+user_base_impact: medium
+
+tags:
+  - css
+  - float
+
+symptoms:
+    - "Attempting to check a custom styled checkbox element has no effect"
+    - "Text content or element is shifted to the right, possibly invisible"
+
+solutions:
+  interventions:
+    - https://github.com/mozilla-extensions/webcompat-addon/blob/main/src/injections/css/bug1765947-veniceincoming.com-left-fix.css
```

Comment:
We really should use the canonical URL form (i.e. press the `y` key while on that page). Having it link to just the `main` branch is planting a ticking time bomb - that link will 404 as soon as we remove the intervention (or rename it)

Reply:
https://github.com/webcompat/knowledge-base/issues/72 for adding a lint here, although if we could handle renames, "this intervention was removed (or renamed)" seems like useful data; we probably shouldn't be linking to interventions that aren't shipping anymore. Alternatively, if we think "there was an intervention for this" is useful historical data then we should make it possible to distinguish shipping and unshipped interventions. (this comment should now probably be a second issue :) ).

Reply:
>  if we think "there was an intervention for this" is useful historical data then we should make it possible to distinguish shipping and unshipped interventions

I think it undoubtedly is. If a site has been fixed and we removed the intervention, we might run into the same issue on another site, and it would be amazing to have a shortcut to the intervention.

Reply:
Thanks! I've changed the URLs.